src/core/agent/services/signifyNotificationService.test.ts

```diff
@@ -277,4 +310,154 @@ describe("Signify notification service of agent", () => {
       signifyNotificationService.unreadNotification("not-exist-noti-id")
     ).rejects.toThrowError(SignifyNotificationService.NOTIFICATION_NOT_FOUND);
   });
+
+  test("Should skip if there is a missing multi-sig identifier", async () => {
+    const callback = jest.fn();
+    notificationStorage.findAllByQuery = jest.fn().mockResolvedValue([]);
+    groupGetRequestMock.mockResolvedValue([{ exn: { a: {} } }]);
+    identifierStorage.getIdentifierMetadata = jest.fn().mockResolvedValue({});
+    const notes = [
+      {
+        i: "string",
+        dt: "string",
+        r: false,
+        a: {
+          r: "/multisig/rpy",
+          d: "not-found-said",
+          m: "",
+        },
+      },
+    ];
+    for (const notif of notes) {
+      await signifyNotificationService.processNotification(notif, callback);
+    }
+    expect(callback).toBeCalledTimes(0);
+  });
+
+  test("Should skip if notification route is rpy and identifier authorizedEids includes eid", async () => {
```

Comment:
Hey @Sotatek-BaoHoanga yesterday we removed authorizedEids as it was causing an issue, so it's been simplified. Can you update these tests?